### DISCUSSION TITLE
SerializeField가 안됩니다.

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#573] [Issue Report (Legacy)]
SerializeField로 다른 스크립트를 가져오려고 하는데, 해당 조작이 안됩니다.

![image](https://user-images.githubusercontent.com/5895396/161485250-e7be9133-cb2e-4844-91d0-7aa71a445970.png)

![image](https://user-images.githubusercontent.com/5895396/161485274-d25048a3-1b6a-4598-98b5-4cbb48a1d519.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 관련 내용 재현이 되지 않아서 개발중인 환경 및 월드 버전을 올려주시기를 부탁드립니다.

![Screen Shot 2022-04-04 at 8 04 30 PM](https://user-images.githubusercontent.com/100118650/161531162-61a397fa-249e-4f68-af70-e27ea6fc0d6b.png)

![Screen Shot 2022-04-04 at 8 04 18 PM](https://user-images.githubusercontent.com/100118650/161531149-2d996750-dac6-4d61-8cb9-620e44d4f73f.png)

**Reply by ryusiin** [reply]
ZEPETO World Version
![image](https://user-images.githubusercontent.com/5895396/161662379-7ad64425-37c5-4f89-bc6e-dea958248bba.png)

개발 환경 : Windows 10

=====

죄송합니다, 조금 설명이 부족했던 듯 한데
number, string 등의 기본적인 타입의 SerializeField는 작동을 하는데, 위와 같이 직접 작성한 ZepetoScript 등의 SerializeField가 작동하지 않는 현상입니다.

**Reply by Zepeto-TechEvangelist** [reply]
현재 제페토 스크립트에서 커스텀 클래스는 지원하지 않습니다. 지원 계획은 내부에서 검토 중에 있는점 참고 부탁 드립니다.

현재 지원하는 부분은 아래 가이드에 있는 내용을 참고하여 주세요.
https://studio.zepeto.me/guides/properties

**Reply by minjujuu** [reply]
직접 작성한 ZepetoScript를 불러와서 사용하시려면,
해당 스크립트가 붙어있는 오브젝트를 인스펙터에서 받아와서 GetComponent를 통해 사용하실 수 있을 것 같습니다!
```
         public customClassObject: GameObject;
         private customClass: CustomClass;

         Start() {
                 this.customClass = this.customClassObject.GetComponent<CustomClass>();
          }
```
